webserver/static/js/eval-jobs-viewer.js

```diff
@@ -237,10 +239,22 @@ var JobDetails = React.createClass({
                 </div>;
                 break;
             case JOB_STATUS_DONE:
-                status = <div className="alert alert-success">
-                    This evaluation job has been completed on {this.props.updated}.
-                    You can find results below.
-                </div>;
+		
+		switch (this.props.valid) {
+		case '1':
```

Comment:
see the way we do the job status above, we define some constants, e.g. `JOB_STATUS_DONE`, so that it's easier to read. You could do the same here - `JOB_DATASET_VALID`, `_NOTVALID`?
I'm not sure valid is the terminology we want to use here. I guess you want to say "the results are still valid", but the word is a little confusing because it can also mean things like "not made correctly". What about using `OUTDATED`?